src/main/java/at/hannibal2/skyhanni/features/garden/pests/PestType.kt

```diff
@@ -93,21 +106,116 @@ enum class PestType(
         "PEST_SLUG_MONSTER".toInternalName(),
         CropType.MUSHROOM,
     ),
-    MOUSE(
-        "Field Mouse",
-        BossType.GARDEN_PEST_MOUSE,
+    // For use in the Pest Profit Tracker, in cases where an item cannot have an identified PestType
+    // Display name intentionally omitted to aid in filtering out this entry.
+    UNKNOWN(
```

Comment:
can we use nullable instead?

Reply:
We can't use null, no - null is the BucketTracker way of saying that data is coming from all buckets, and the maps it defines are around Enum<E>, which shouldn't be nullable type keys